thebeetoken/beenest-web
# Add QR code for BTC payment

## Description
Adds a QR code to simplify Bitcoin payments

Note that this depends on #79 

## How to Test
1. Make a BTC booking
2. Verify that a QR code for payment is displayed

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
BTC handling for host approval

## Learnings
There's a [Bitcoin URI format](https://en.bitcoin.it/wiki/BIP_0021) which includes useful fields like `amount`
